src/app/state/app.selectors.ts

```diff
@@ -8,18 +8,9 @@ export namespace AppSelectors {
   export const getProjectsState = createFeatureSelector<ProjectsState>('projects');
   export const getRouterReducerState = createFeatureSelector<RouterReducerState>('router');
 
-  export const getRouterState = createSelector(
-    getRouterReducerState,
-    router => router.state
-  );
+  export const getRouterState = createSelector(getRouterReducerState, router => router.state);
 
```

Comment:
Now that they are in a single line let's remove the newlines between these consts.